mailipy/send.py

```diff
@@ -95,7 +97,9 @@ def send_emails(server, emails, outbox_dir, sent_dir):
 
             # On success, move the message from the outbox to the sent folder
             shutil.move(os.path.join(outbox_dir, eml), os.path.join(sent_dir, eml))
-        except:
+            time.sleep(int(args.sleep))
```

Comment:
It would be nicer to avoid calling sleep() at all when the value of args.sleep is 0.

```suggestion
            if args.sleep > 0:
                time.sleep(args.sleep)
```